storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/integration/GetOperationTest.java

```diff
@@ -165,4 +165,34 @@ public void getListOfObjectsAsObservableOnlyInitialValue() {
         changesTestSubscriber.assertNoErrors();
         changesTestSubscriber.assertValues(Changes.newInstance(TestItem.CONTENT_URI));
     }
+
+    @Test
+    public void getNumberOfResults() {
+        TestSubscriber<Changes> changesTestSubscriber = new TestSubscriber<Changes>();
+
+        storIOContentResolver
+                .observeChangesOfUri(TestItem.CONTENT_URI)
+                .take(1)
```

Comment:
You should take 8 changes here because you're inserting 8 items separately and ContentProvider throws 8 notifications.

That was the problem of our tests flakiness :)